### PR TITLE
Add model -> user

### PR DESCRIPTION
**Pull request by flavv85**

### CODE REVIEW

File: src/main/java/model/User.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package model;
+
+public class User {
+}
```
Comment (stefan2212) [inline src/main/java/model/User.java]: Empty line between {}

**Review by stefan2212 — CHANGES_REQUESTED** [review]

File: src/main/java/model/User.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package model;
```
Comment (RobertChipaila) [inline src/main/java/model/User.java:1]: package e bine

**Review by stefan2212 — APPROVED** [review]
LGTM

**Comment by flavv85** [comment]
Rezolvat

**Pull request merged**